common/src/main/java/com/tc/net/core/BufferManager.java

```diff
@@ -27,12 +27,40 @@
  * @author Ludovic Orban
  */
 public interface BufferManager {
+  /**
+   * Called immediately after a call to recvToBuffer(). It will be called again if it returns a non-zero value.
```

Comment:
`It will be called again if it returns a non-zero value.` states an implementation detail of a consumer of this interface, namely the comm stack.